modules/core/03-connection/keeper/verify.go

```diff
@@ -28,12 +32,30 @@ func (k Keeper) VerifyClientState(
 	}
 
 	if status := targetClient.Status(ctx, clientStore, k.cdc); status != exported.Active {
-		return sdkerrors.Wrapf(clienttypes.ErrClientNotActive, "client (%s) status is %s", clientID, status)
+		return sdkerrors.Wrapf(clienttypes.ErrClientStateNotFound, "client (%s) status is %s", clientID, status)
 	}
 
-	if err := targetClient.VerifyClientState(
-		clientStore, k.cdc, height,
-		connection.GetCounterparty().GetPrefix(), connection.GetCounterparty().GetClientID(), proof, clientState); err != nil {
+	merklePath := commitmenttypes.NewMerklePath(host.FullClientStatePath(connection.GetCounterparty().GetClientID()))
+	merklePath, err := commitmenttypes.ApplyPrefix(connection.GetCounterparty().GetPrefix(), merklePath)
```

Comment:
there shouldn't be an issues with reusing the same var and `:=` right?

Reply:
yep, the second `:=` short-hand assignment should be for initialising the `err` return and re-assign the `merklePath`